public/app/features/explore/Logs/Logs.tsx

```diff
@@ -283,6 +283,10 @@ class UnthemedLogs extends PureComponent<Props, State> {
     if (this.props.onStartScanning) {
       this.props.onStartScanning();
     }
+    reportInteraction('grafana_explore_logs_scanning_button_clicked', {
+      type: 'start',
+      datasourceType: this.props.datasourceType,
+    });
   };
```

Comment:
Shouldn't this be:

```suggestion
    if (this.props.onStartScanning) {
      this.props.onStartScanning();
      reportInteraction('grafana_explore_logs_scanning_button_clicked', {
        type: 'start',
        datasourceType: this.props.datasourceType,
      });
    }
  };
```

Reply:
hmmm.. good question.. i guess the question is "if the user presses the button, but there's no handler, should we count that?" .. i'll check the code more to see why that prop is optional.

Reply:
i applied the change.